docs/user_guide/alerts_dashboards.md

```diff
@@ -224,62 +159,200 @@ There are several security related reports to choose from and they are all locat
+    * Authentication - Excessive Failed VPN Logins from a Source
+    * Authentication - Long Running VPN Session Disconnected
+    * Authentication - Successful VPN Login From New Location
+    * Authentication - Successful VPN Login Outside Home Country
+    * Authentication - VPN Login Attempts Outside Working Hours
+* Fortinet FortiGate Dashboards:
     * Connected Workforce by Location
     * Elapsed Time Per Session
+    * Login Details
     * Logins by Country
-    * Logins by Success/Failure
+    * Logins by Success/Failure 
+    * Successful Session
+    * Successful vs Failed Logins
+    * Unique Users by Country
+* GlobalProtect (Palo Alto) Alerts:
+    * Authentication - Bruteforce Attempt for a User
+    * Authentication - Bruteforce Attempt from a Source
+    * Authentication - Excessive Failed VPN Logins for a User
+    * Authentication - Excessive Failed VPN Logins from a Source
+    * Authentication - Long Running VPN Session Disconnected
+    * Authentication - Successful VPN Login From New Location
+    * Authentication - Successful VPN Login Outside Home Country
+    * Authentication - VPN Login Attempts Outside Working Hours
+* GlobalProtect (Palo Alto) Dashboards:
+    * Connected Workforce by Location
+    * Elapsed Time Per Session
     * Login Details
+    * Logins by Country
+    * Logins by Success/Failure 
     * Successful Session
     * Successful vs Failed Logins
     * Unique Users by Country
 
-* Windows Reports
-    * Operation Attempted on Privileged Object (EventCode=4674)
-    * Privileged Service Accessed (EventCode=4673)
-    * Windows Users and Privileges (EventCode=4672)
+## Vulnerability 
 
-* Windows Defender Reports 
-    * Antimalware Dropped Support for OS or Antimalware Engine Dropped Support for OS
-    * Antimalware will Drop Support for Operating System Soon 
-    * Antimalware will Expire Soon 
-    * Antivirus Expired 
-    * Antivirus Scan Failed 
-    * Malware Detected 
-    * Unable to Download and Configure Offline Scan 
-    * Update Errors
-    * Windows Defender Health Report 
+Supported vendor products include: CrowdStrike Spotlight, Nessus, Qualys & Tenable IO
+
+These vendor security solutions are designed to detect vulnerabilities present in your environment.  
+
+The Cyences app utilizes the data provided by the aforementioned vendor products to obtain information pertaining to any vulnerabilities that may exist on an IT asset within your environment.
+
+The Vulnerability dashboard is designed to view the vulnerability count by severity, new vulnerabilities found over time, a vulnerability summary based on host(s), and a list of vulnerabilities. Splunk users can further use the drilldown capability to view a vulnerability list for a single host or a list of hosts affected by a vulnerability.
+
+
+![alt](https://github.com/CrossRealms/Splunk-Cyences-App-for-Splunk/blob/master/docs/assets/vulnerability_dashboard.png?raw=true)
+
+
+* CrowdStrike Spotlight Alerts:
```

Comment:
No need to specify each separately. Just specify:
* Dashboard panels:
* * New Vulnerability Found Over Time
    * Vulnerabilities 
    * Vulnerability Count By Severity